dep/src/main/java/dep/badClassName.java

```diff
@@ -0,0 +1,19 @@
+package dep;
+
+public class badClassName {
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1135)